### PR TITLE
Add bitfields and anon containers

### PR DESCRIPTION
**Pull request by roblabla**
https://github.com/PrismarineJS/minecraft-data/pull/63

### CODE REVIEW

File: src/datatypes/minecraft.js [hunk 1]
```diff
@@ -25,35 +24,6 @@ function writeUUID(value, buffer, offset) {
...
-  var longVal = numeric.long[0](buffer, offset).value;
-  var x = signExtend26(longVal[0] >> 6);
-  var y = signExtend12(((longVal[0] & 0x3f) << 6) | ((longVal[1] >> 26) & 0x3f));
-  var z = signExtend26(longVal[1] & 0x3FFFFFF);
-  return {
-    value: {x: x, y: y, z: z},
-    size: 8
-  };
-}
-function signExtend26(value) {
```
Comment (rom1504) [inline src/datatypes/minecraft.js]: Is this handled by the bitfield type ?

Comment (roblabla) [inline src/datatypes/minecraft.js]: Yup.

Comment (rom1504) [inline src/datatypes/minecraft.js]: I meant the change of sign

Comment (roblabla) [inline src/datatypes/minecraft.js]: ah, nicely spotted, probably not. I'll need to add a signed: true option to the bitfields I guess

**Comment by rom1504** [comment]
This requires:
1. Fixing numerical packets (related to #210 ) : related with #212 
2. Updating node-minecraft-data

Then it'll be ready to merge

**Comment by roblabla** [comment]
This doesn't require fixing the numerical packets. It's just waiting for an NMD update. Fixing numerical packets is required for the 1.9 patch.

**Pull request merged**